internal/pkg/deploy/cloudformation/stack/workload.go

```diff
@@ -111,6 +111,12 @@ type location interface {
 	GetLocation() string
 }
 
+// uploadable is the interface for an object that can be uploaded to an S3 bucket.
```

Comment:
Is the plan to do something similar to ⬇️  in `Template()`:
```go
func (svc *BackendService) Template() (string, error) {
   for _, uploadable := range svc.localCRs {
      url, err := s3.upload(uploadable.ArtifactPath(), uploadable.Content())
      crURLs[uploadable.Name()] = url
   }
  opt := &template.WorkloadOpts{
      customResources: crURLs
  }
}
```

Reply:
yup!